go/vt/sqlparser/parse_test.go

```diff
@@ -1679,11 +1685,69 @@ var (
 		}, {
 			input:  "delete a.*, b.* from tbl_a a, tbl_b b where a.id = b.id and b.name = 'test'",
 			output: "delete a, b from tbl_a as a, tbl_b as b where a.id = b.id and b.name = 'test'",
+		}, {
+			input: "call f1",
+		}, {
+			input:  "call f1()",
+			output: "call f1",
+		}, {
+			input:  "call f1 ()",
+			output: "call f1",
+		}, {
+			input: "call f1(x)",
+		}, {
+			input: "call f1(@x, @y)",
+		}, {
+			input: "call f1(now(), rand())",
+		}, {
+			input: "drop procedure p1",
+		}, {
+			input: "drop procedure if exists p1",
+		},
+	}
+	// Any tests that may contain multiple statements within the body (such as BEGIN/END blocks) should go here.
+	// validSQL is used by TestParseNextValid, which expects a semicolon to mean the end of a full statement.
+	// For convenience, tests of the same statement, even when not in a multi-statement form, can go here.
```

Comment:
Seems like most of these test cases don't match this statement. Is this more about needing to match without exact formatting (for newlines)?

If so, you should be able to move most of these to the main block above.